### PR TITLE
Feat/379 nodejs framework data

### PR DESCRIPTION
**Pull request by morgnism**
## Type of change

- [x] Content addition
- [ ] Bug fix
- [ ] Behavior change

## Checklist

- [x] The changes follow the [contributing guidelines](https://github.com/thisdot/framework.dev/blob/main/CONTRIBUTING.md)
- [x] I have searched all existing content and verified my additions are novel
      and unique
- [x] The content was added to the end of the appropriate data list
- [x] All required content fields are accurately populated
- [x] All items are tagged with all relevant tags

### CODE REVIEW

**Review by jdwilkin4 — APPROVED** [review]
I reviewed in locally and everything looks good.
Just pushed up a small change to add an image to the nodejs blog 👍

File: packages/site/src/data/nodejs/libraries.ts [hunk 1]
```diff
@@ -1,26 +1,166 @@
...
 	"data fetching",
-	"GraphQL",
-	"Utility-First CSS",
-	"CSS in JS",
-	"component library",
-	"bootstrap",
-	"forms",
-	"reactive forms",
 	"testing",
+	"orm",
```
Comment (jesus4497) [inline packages/site/src/data/nodejs/libraries.ts:10]: I think we would like this to be `ORM` instead

File: packages/site/src/data/nodejs/blogs.ts [hunk 2]
```diff
@@ -2,4 +2,72 @@ import { Blog } from "@framework/system/src/models/blog"
 
 export const blogTags = [] as const
 
-export const blogs: Blog<typeof blogTags[number]>[] = []
+export const blogs: Blog<typeof blogTags[number]>[] = [
+	{
+		title: "This Dot Blog",
+		author: "This Dot Labs",
+		description:
+			"Variety of topics related to Angular, written by the team at This Dot",
```
Comment (jesus4497) [inline packages/site/src/data/nodejs/blogs.ts:10]: Is this blog meant to be here? Seems like an angular one

File: packages/site/src/data/nodejs/blogs.ts [hunk 3]
```diff
@@ -2,4 +2,72 @@ import { Blog } from "@framework/system/src/models/blog"
...
+		href: "https://dev.to/t/node",
+		tags: [],
+	},
+	{
+		title: "Kent C. Dodds' Blog",
+		author: "Kent C. Dodds",
+		description:
+			"Renowned JavaScript and React educator, Kent C. Dodds, shares his React knowledge through his blog.",
+		image: "https://github.com/kentcdodds.png",
+		href: "https://kentcdodds.com/blog?q=react",
```
Comment (jesus4497) [inline packages/site/src/data/nodejs/blogs.ts:41]: This seems to be pointing to the react part of the blog

File: packages/site/src/data/nodejs/blogs.ts [hunk 4]
```diff
@@ -2,4 +2,72 @@ import { Blog } from "@framework/system/src/models/blog"
...
+		href: "https://wesbos.com/blog",
+		tags: [],
+	},
+	{
+		title: "Overreacted",
+		author: "Dan Abramov",
+		description: "Personal developer blog of React creator, Dan Abramov.",
+		image:
+			"https://overreacted.io/static/profile-pic-c715447ce38098828758e525a1128b87.jpg",
+		href: "https://overreacted.io/",
```
Comment (jesus4497) [inline packages/site/src/data/nodejs/blogs.ts:70]: Might be wrong! but I'm not seeing any node related post here

File: packages/site/src/data/nodejs/libraries.ts [hunk 5]
```diff
@@ -1,26 +1,166 @@
 import { Library } from "@framework/system/src/models/library"
 
 export const libraryTags = [
-	"state management",
-	"internationalization",
 	"framework",
 	"documentation",
-	"styling",
-	"component",
 	"logging",
```
Comment (jesus4497) [inline packages/site/src/data/nodejs/libraries.ts:6]: When tend to use `authentication` for this tag

File: packages/site/src/data/nodejs/blogs.ts (hunk 2 above)
Comment (morgnism) [reply packages/site/src/data/nodejs/blogs.ts:10]: Yes, we want the This Dot blog, but it should reference Node instead.

File: packages/site/src/data/nodejs/libraries.ts (hunk 5 above)
Comment (morgnism) [reply packages/site/src/data/nodejs/libraries.ts:6]: Do you mean `authentication` in place of logging?

I can add that as a tag, but there aren't specific node libs that focus on authentication. And there are libs specifically for logging in node (planning to add more libs featuring those in another PR).

Comment (jesus4497) [reply packages/site/src/data/nodejs/libraries.ts:6]: Oh! then `logging` is fine 🚀

File: packages/site/src/data/nodejs/blogs.ts (hunk 2 above)
Comment (morgnism) [reply packages/site/src/data/nodejs/blogs.ts:10]: Looking at it again, we don't have Node-specific articles yet, but Dustin wants the This Dot blog included anyway.

**Review by jesus4497 — APPROVED** [review]

**Pull request merged**